src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+const roundNumber = (num: number): number => {
+  return Math.floor(num * 100) / 100;
+};
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  constructor(
+    public color: Figure['color'],
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
```

Comment:
After making a `Shape` enum
```suggestion
    this.shape = Shape.triangle;
```